src/option_parser.c

```diff
@@ -198,14 +198,11 @@ char *clean_value(char *value, int line_num)
                         in_quote = !in_quote;
                         break;
                 case '\\':
-                        switch (unparsed[1]) {
-                        case '\\':
-                        case '"':
-                                memmove(unparsed, unparsed + 1, strlen(unparsed));
-                                unparsed++;
-                                break;
+                        memmove(unparsed, unparsed + 1, strlen(unparsed));
+                        switch (*unparsed) {
+                        case '\\': /* verbatim */ break;
+                        case '"': /* verbatim */ break;
```

Comment:
To improve readability, there shouldn't be anything else on a line with a switch case label. I think it's best to utilize fall-through on this as it was before.
```
case '\\':
case '"':
        break;
```